Plots/Satellites/NCL_sat_2.py

```diff
@@ -0,0 +1,436 @@
+
+        clusterMins.append((lonvalue, latvalue))
+
+    return clusterMins
+
+###############################################################################
+# Helper function that finds the local high pressure coordinates
+# on a contour map.
+
+
+def findLocalMaxima(maxPressure=1040):
+
+    # Create a 2D array of all the coordinates with pressure data
+    coordarr = makeCoordArr()
+
+    # Set number that a derivative must be less than in order to
+    # classify as a "zero"
+    bound = 0.0
+
+    # Get global gradient of contour data
+    grad = np.gradient(wrap_pressure.data)
+
+    # Gradient in the x direction
+    arr1 = grad[0]
+
+    # Gradient in the y direction
+    arr2 = grad[1]
+
+    # Get all array 1 indexes where gradient value is between -bound and +bound
+    posfirstzeroes = np.argwhere(arr1 <= bound)
+    negfirstzeroes = np.argwhere(-bound <= arr1)
+
+    # Get all array 2 indexes where gradient value is between -bound and +bound
+    possecondzeroes = np.argwhere(arr2 <= bound)
+    negsecondzeroes = np.argwhere(-bound <= arr2)
+
+    # Find zeroes of all four gradient arrays
+    commonzeroes = []
+    for x in possecondzeroes:
+        if x in posfirstzeroes and x in negfirstzeroes and x in negsecondzeroes:
+            commonzeroes.append(x)
+
+    maximacoords = []
+    coordsinthearray = []
+
+    # For every common zero in both gradient arrays
+    for x in commonzeroes:
+
+        try:
+            xval = x[0]
+            yval = x[1]
+
+            # If the gradient value is a "zero", and if the
+            # pressure.data value is greater than maxPressure:
+            if wrap_pressure.data[xval][yval] >= maxPressure:
+
+                coordonmap = coordarr[xval][yval]
+
+                coordsinthearray.append((xval, yval))
+
+                # Get coordinate from index in coordArr
+                xcoord = coordonmap[0]
+                ycoord = coordonmap[1]
+
+                maximacoords.append((xcoord, ycoord))
+        except:
+            continue
+
+    coordsAndLabels = getKClusters(maximacoords, numClusters=3)
+    clusterMaxs = findClusterMax(coordarr, coordsAndLabels)
+
+    return clusterMaxs
+
+
+###############################################################################
+# Create plot
+
+# Set figure size
+fig = plt.figure(figsize=(8, 8))
+
+# Set global axes with an orthographic projection
+proj = ccrs.Orthographic(central_longitude=270, central_latitude=45)
+ax = plt.axes(projection=proj)
+ax.set_global()
+
+# Add land, coastlines, and ocean features
+ax.add_feature(cfeature.LAND, facecolor='lightgray', zorder=1)
+ax.add_feature(cfeature.COASTLINE, linewidth=.3)
+ax.add_feature(cfeature.OCEAN, facecolor='white')
+ax.add_feature(cfeature.BORDERS, linewidth=.3)
+ax.add_feature(cfeature.LAKES, facecolor='white',
+               edgecolor='k', linewidth=.3)
+
+# Make array of the contour levels that will be plotted
+contours = np.arange(948, 1060, 4)
+contours = np.append(contours, 975)
```

Comment:
Do you still need to explicitly add this contour level? There aren't any low pressure contours that need to be drawn with a value of 975.
Additionally, would it be possible to add to the helper methods a way to draw the contours that aren't already drawn if they need a high or low pressure label? That way you wouldn't have to manually append the value to this array.